yvalenta/lib/client/boot.rb

```diff
@@ -0,0 +1,3 @@
+require 'client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.